tools/schemacode/setup.cfg

```
@@ -0,0 +1,68 @@
+[metadata]
+name = schemacode
+url = https://github.com/bids-standard/schemacode
+author = bids-standard developers
+author_email = bids.maintenance@gmail.com
+description = Python tools for working with the BIDS schema.
+long_description = file:README.md
+long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
+license = MIT
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering :: Information Analysis
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
```

what about
```suggestion
    Programming Language :: Python :: 3.9
    Programming Language :: Python :: 3.10
```
?
versioneer 0.21 is compatible according to changes done since 0.21 (just addition of 3.10 marker)